src/main/java/ru/testtask/service/ProjectService.java

```diff
@@ -30,13 +38,20 @@ public Project findProjectByName(String name){
     }
 
 
-    public Project createProject(Project project) {
-        for (int i = 0; i < 10; i++) {
-            project.addGeometry(Geometry.builder().name("geometry" + i).id(UUID.randomUUID().toString()).build());
-            project.addAttribute(Attribute.builder().name("geometry" + i).id(UUID.randomUUID().toString()).build());
+    public Project createProject(Project project) throws NameAlreadyExistsException {
+        if (findProjectByName(project.getName()).getId() != null){
```

Comment:
I think this code will work only in case if you try to create new project with conflict name (if project with same name already exists)
`findProjectByName(project.getName())` will return `null` in any other cases